runtime/compiler/p/codegen/PPCJNILinkage.cpp

```diff
@@ -63,6 +63,8 @@ J9::Power::JNILinkage::JNILinkage(TR::CodeGenerator *cg)
 
    _properties._preservedRegisterMapForGC = privateLinkageProperties.getPreservedRegisterMapForGC();
 
+   // TODO (#9999): JNI linkage should inherit from system linkage to avoid having to do this copying
+   self()->setOffsetToFirstParm(sysLinkage->getOffsetToFirstParm());
```

Comment:
Noting this for myself to fix. I'll need to open up an issue and correct this reference above. This is a particularly tricky thing we do on Power due to the way linkage properties work and the fact that JNI linkage inherits from Private linkage.